constants/labels.en.js

```diff
@@ -16,6 +16,19 @@ const labels = {
         description:
             "Join a network that connects you to a IT community expanding your horizons and unlocking new opportunities. Connect with a vibrant community of tech-savvy individuals and be in the know of the latest happenings within Brito. Elevate your skills and join a network that's dedicated to fostering growth, innovation and collaboration!",
     },
+
+    welcomeToBrito: {
+        title: 'WELCOME TO BRITO',
+        description: [
+            'BRAZILIAN IT OTTAWA is non-profit group of IT Brazilians professionals located in Ottawa.',
+            <br />,
+            'Our main objective is empowering the community, assisting with networking, and continuous' +
+                'learning with lectures and workshops.'
+        ],
+        img_source:'/ellipse.png',
```

Comment:
Please usa an image object

Reply:
Passed as an object. Thanks for showing me how to do it!